pkg/api/types.go

```diff
@@ -360,7 +360,7 @@ type PersistentVolumeSource struct {
 	Cinder *CinderVolumeSource
 	// CephFS represents a Ceph FS mount on the host that shares a pod's lifetime
 	// +optional
-	CephFS *CephFSVolumeSource
+	CephFS *CephFSPersistentVolumeSource
```

Comment:
This is IMO API breaker. My yaml file with Ceph PersistentVolume that was working in 1.7 won't work in 1.8. Also what happens to my  PersistentVolume stored in etcd? How it gets upgraded from CephFSVolumeSource to CephFSPersistentVolumeSource?